src/ert/trace.py

```diff
@@ -9,3 +9,7 @@
 trace.set_tracer_provider(tracer_provider)
 
 tracer = trace.get_tracer("ert.main")
+
+
+def getTraceID() -> str:
```

Comment:
rename to  `get_trace_id()` for consistent style?

Reply:
👌